Makefile

```diff
@@ -40,7 +40,7 @@ install:
 
 install_lint:
 	@echo "Installing golint, since we expect to lint"
-	go get -u -f golang.org/x/lint/golint
+	go install golang.org/x/lint/golint@latest
```

Comment:
If you do `go install <pkg>`, it'll use a version specified in the go.mod, which allows the tools to be version-managed like other deps (more consistent experience that doesn't depend on the host env).

I'd suggest making the tools managed more like zap,
 * Versioned, but separate module (so deps for the tools don't affect deps for prod)
 * Easy to install to a location like ./bin (see `GOBIN`)
 * More consistency (think atomic is the same).

Reply:
great suggestion. i'll look into that. thanks!